### PR TITLE
[cherry-pick in v6r15] Take into account WallTime during MaxCPUTime definition

### PR DESCRIPTION
**Pull request by VladimirRomanovsky**
Minot change to BDII2CS

### CODE REVIEW

**Comment by fstagni** [comment]
This one is targeting integration. @andrei maybe can cherry-pick the commit in v6r15

File: ConfigurationSystem/Client/Utilities.py [hunk 1]
```diff
@@ -273,6 +273,13 @@ def addToChangeSet( entry, changeSet ):
           newMaxCPUTime = queueInfo.get( 'GlueCEPolicyMaxCPUTime', '' )
           if newMaxCPUTime == "4" * len( newMaxCPUTime ) or newMaxCPUTime == "9" * len( newMaxCPUTime ):
             newMaxCPUTime = ''
```
Comment (fstagni) [inline ConfigurationSystem/Client/Utilities.py]: I know the 2 lines above are not part of the changes of this PR, but... what the heck is that?

File: ConfigurationSystem/Client/Utilities.py [hunk 2]
```diff
@@ -273,6 +273,13 @@ def addToChangeSet( entry, changeSet ):
...
+          wallTime = queueInfo.get( 'GlueCEPolicyMaxWallClockTime', '' )
+          if wallTime and int(wallTime)>0:
+            if not newMaxCPUTime:
+              newMaxCPUTime = str(0.8*int(wallTime))
+            else:
+              if int(wallTime) <= int(newMaxCPUTime):
+                newMaxCPUTime = str(0.8*int(wallTime))
```
Comment (fstagni) [inline ConfigurationSystem/Client/Utilities.py]: In general, why applying an 80% factor?

File: ConfigurationSystem/Client/Utilities.py (hunk 1 above)
Comment (sposs) [inline ConfigurationSystem/Client/Utilities.py]: You know there was a controversy with respect to 'default' values in the Glue Schema at some point. Those lead to interesting discrepancy between the OSG and LCG... Basically, some think that putting 99999 or 44444 as default is reasonable. Those mean there are no known values. It's awful and breaks the OSG brokers...

File: ConfigurationSystem/Client/Utilities.py (hunk 2 above)
Comment (phicharp) [inline ConfigurationSystem/Client/Utilities.py]: If the wall time limit is shorter or equal to the CPU time limit, this is a bad idea from he site... Assume jobs have more than 80% CPU efficiency and get the CPU time limit hitting first

**Comment by VladimirRomanovsky** [comment]
The purpose of the PR is to cover two really existing cases:
1. Some site have no any limit on CPU time and put there huge number (maxint?).
2. Other sites define CPU time as 16*Walltime arguing that this queue will be used for multiprocessing jobs. The site from first case also mentioned multiprocessing job.

All these sites strongly object against changing CPU time.

**Comment by fstagni** [comment]
So, what's the review of this? I guess "OK", but I would ask for some explanation in the code, because all this stuff would be easily forgot next time.

**Comment by phicharp** [comment]
For me it is OK

**Pull request merged**